example/server/integrations.py

```diff
@@ -970,3 +988,161 @@ def info_integration(request: Request, asset: Asset, lang: str):
             }
         },
     }
+
+
+def check_delivery_method(
+    asset: str,
+    list_delivery_methods_func: Callable[[OffChainAsset], List],
+    delivery_method: str,
+) -> bool:
+    if delivery_method:
+        bdms = list_delivery_methods_func(get_offchain_asset(asset))
+        if not any(bdm.name == delivery_method for bdm in bdms):
+            # This method is not supported
+            return False
+    return True
+
+
+def check_country_code(asset: str, country_code: str) -> bool:
+    if country_code and country_code not in get_offchain_asset(
+        asset
+    ).country_codes.split(","):
+        # The contry code is not supported by the anchor.
+        return False
+    return True
+
+
+class MySEP38AnchorIntegration(SEP38AnchorIntegration):
+    def get_prices(
+        self,
+        sell_asset: str,
+        sell_amount: str,
+        sell_delivery_method: str = None,
+        buy_delivery_method: str = None,
+        country_code: str = None,
+    ) -> List[GetPricesResponse]:
+        exchange_pairs = list_exchange_pairs(buy_asset=sell_asset)
+
+        prices = []
+        for exchange_pair in exchange_pairs:
+            if is_stellar_asset(sell_asset):
+                # We are selling a stellar asset. Buying asset is offchain.
```

Comment:
In the API, `sell_asset` is defined as the asset the client wants to sell. So if `is_stellar_asset(is_sell_asset)` is `True`, that means the anchor is buying a stellar asset, and selling an off-chain asset.